sdk/formrecognizer/Azure.AI.FormRecognizer/src/AccountProperties.cs

```diff
@@ -29,7 +29,7 @@ internal AccountProperties(int customModelCount, int customModelLimit)
         }
 
         /// <summary>
-        /// The current count of trained custom models.
+        /// The current count of trained custom models in this subscription.
```

Comment:
should it say in this subscription here? Or is this per account?

Reply:
ohh you right. account is more appropriate. I will fix